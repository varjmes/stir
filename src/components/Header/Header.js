import React from 'react'

export default () => <h1>stir.</h1>