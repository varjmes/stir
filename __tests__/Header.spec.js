import React from 'react'
import renderer from 'react-test-renderer'

import Header from '../src/components/Layout/Header/Header'

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Header />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
