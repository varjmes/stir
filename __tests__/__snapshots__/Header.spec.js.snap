// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<h1
  className="sc-bdVaJa LoiiH"
>
  stir.
</h1>
`;
